apps/frontend/src/components/review/TechnicalBulkReassignModal.tsx

```diff
@@ -0,0 +1,301 @@
+    },
+  ];
+
+  return (
+    <>
+      <StyledDialog
+        aria-labelledby="simple-modal-title"
+        aria-describedby="simple-modal-description"
+        open={true}
+        onClose={(): void => setReviews()}
+        maxWidth="lg"
+        fullWidth
+        title={`Bulk Reassign Proposals Techniqual reviews`}
+        data-cy="bulk-reassigne-modal"
+      >
+        <DialogContent>
+          <FormHelperText error></FormHelperText>
+          {!!proposalsWithMultipleInstrument.length && (
+            <Alert severity="warning" data-cy="multi-instrument-alert">
+              {`Warning you have selected proposal(s) with multiple ${t('instrument')}s then
+              both there reviews will apear here: ${proposalsWithMultipleInstrument.toString().replaceAll(',', ', ')}`}
+            </Alert>
+          )}
+          <MaterialTable
+            title={
+              <Typography variant="h6" component="h1">
+                Techniqual Reviews
+              </Typography>
+            }
+            columns={instrumentColumns}
+            data={reviewsSortedByInstrument.map((rev) => {
+              return {
+                ...rev,
+                reviwers: instruments
+                  ? instruments.find((inst) => inst.id === rev.instrumentId)
+                  : null,
+              };
+            })}
+            isLoading={loadingInstruments}
+            options={{
+              debounceInterval: 400,
+              idSynonym: 'instrumentId',
+              search: false,
+            }}
+            detailPanel={[
+              {
+                tooltip: 'Show Manager and Scientists',
+                render: (rowData) => {
+                  return (
+                    <Box
+                      sx={{
+                        '& tr:last-child td': {
+                          border: 'none',
+                        },
+                        '& .MuiPaper-root': {
+                          padding: '0 40px',
+                          backgroundColor: '#fafafa',
+                        },
+                      }}
+                      data-cy={`reassign-proposals-table-${rowData.rowData.instrumentId}`}
+                    >
+                      <MaterialTable
+                        data={rowData.rowData.reviews}
+                        columns={proposalColumns}
+                        options={{
+                          idSynonym: 'review',
+                          search: false,
+                          tableLayout: 'fixed',
+                        }}
+                      />
+                    </Box>
+                  );
+                },
+              },
+            ]}
+          />
+        </DialogContent>
+        <DialogActions>
+          <Button
+            type="button"
+            onClick={() => setReviews()}
+            data-cy="uodate-bulk"
+          >
+            Close
+          </Button>
+          <Button type="button" onClick={updateReviews} data-cy="bulk-update">
```

Comment:
Suggestion: Can we add a confirmation here in the window? Maybe with the Confirm hook